libs/blocks/section-metadata/section-metadata.js

```diff
@@ -34,6 +36,37 @@ function handleTopHeight(section) {
   const headerHeight = document.querySelector('header').offsetHeight;
   section.style.top = `${headerHeight}px`;
 }
+
+export function promoIntersectObserve(el, startPoint, options = {}) {
+  let abovePromoStart = false;
+  const io = new IntersectionObserver((entries, observer) => {
+    entries.forEach(async (entry) => {
+      if (el.classList.contains('close-sticky-section')) return observer.unobserve(entry.target);
+      const isPromoStart = startPoint ? entry.target === startPoint : null;
+      abovePromoStart = isPromoStart
+        ? (entry.isIntersecting || entry.boundingClientRect.y > 0)
+        : abovePromoStart;
+      if (entry.isIntersecting || (!entry.isIntersecting && isPromoStart && abovePromoStart)) el.classList.add('hide-sticky-section');
+      else if (!entry.isIntersecting && !abovePromoStart) el.classList.remove('hide-sticky-section');
+      return el;
```

Comment:
You can remove the return el, it's doing nothing other than making the linter happy.
Either disable the rule or change `if (el.classList.contains('close-sticky-section')) return observer.unobserve(entry.target);` to
```
if (el.classList.contains('close-sticky-section')) {
   observer.unobserve(entry.target);
   return 
}
```

Reply:
@mokimo I have made the requested changes.